bdb/file.c

```diff
@@ -7622,26 +7630,27 @@ int bdb_have_unused_files(void) { return oldfile_list_empty() != 1; }
 int bdb_purge_unused_files(bdb_state_type *bdb_state, tran_type *tran,
                            int *bdberr)
 {
-    char *munged_name = NULL;
-    int rc;
-    unsigned lognum = 0, lowfilenum = 0;
+  char *munged_name = NULL;
```

Comment:
I believe roborivers uses something similar to [this](http://comdb2.s3-website-us-east-1.amazonaws.com/formatting-a-pr.html)